### ISSUE TITLE
Deprecated stuff in Tensorflow

### ISSUE THREAD

**Issue by MatthijsBurgh** [tue-robotics/image_recognition#10]
W tensorflow/core/framework/op_def_util.cc:332] Op BatchNormWithGlobalNormalization is deprecated. It will cease to work in GraphDef version 9. Use tf.nn.batch_normalization().

**Comment by reinzor** [comment]
@Rayman , could you have a look at this?

**Comment by MatthijsBurgh** [comment]
@Rayman status?

**Comment by Rayman** [comment]
I still got not enough time time work on this

**Comment by LoyVanBeek** [comment]
@JosjaG any time to look into this? 
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/graph_transforms/README.md#fold_old_batch_norms might help

**Comment by MatthijsBurgh** [comment]
I don't know if this is still an issue after upgrading to the most recent version of TF.

**Comment by LoyVanBeek** [comment]
Yes, it has only become more urgent. The feature is deprecated and will stop working at a future version and that future is getting closer. 

https://www.tensorflow.org/programmers_guide/version_compat#compatibility_of_graphs_and_checkpoints we're now on TF r1.3 and our network may not be supported anymore on TF r1.4

**Comment by MatthijsBurgh** [comment]
I have done some small research. I think it cause by the pre-trained network, because it is already pretty old.

**Comment by reinzor** [comment]
Let's just put one person on this to fix this instead of heaving these discussions all the time. This is not effective. Let's discuss Tuesday and assign somebody.

**Comment by reinzor** [comment]
According to https://github.com/tensorflow/tensorflow/issues/2164 , we can ignore this
